api/src/functions/messagingService/app.ts

```diff
@@ -103,7 +40,9 @@ const AWSTaxIDEncryptionClient = AWSCryptoFactory(AWS_CRYPTO_TAX_ID_ENCRYPTION_K
 export const handler = async (
   event: SimpleSendRequest,
 ): Promise<{ statusCode: number; messageId: string; body: string }> => {
-  const logger = LogWriter(`HealthCheckService/${STAGE}`, "ApiLogs");
+  // TODO: Need a new log stream for the messaging service
+  const logger = LogWriter(`NavigatorWebService/${STAGE}`, "ApiLogs");
```

Comment:
Currently logging to the same log group as the backend. I think I ultimately want to change that. This is better than us logging to the HealthCheckService.